src/main/java/io/airbrake/log4javabrake2/AirbrakeAppender.java

```diff
@@ -124,7 +128,8 @@ static String formatLevel(Level level) {
   void send(Notice notice) {
     CompletableFuture.runAsync(() -> {
       if (this.notifier != null) {
-        this.notifier.sendSync(notice);
+       this.notifier.sendSync(notice);
+        
```

Comment:
This white space change looks accidental, probably a goodidea to revert for consistent tab use.

Reply:
Ok